2.2/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==19.9.0
-django==2.2
-psycopg2==2.7.7 --no-binary psycopg2
+django==2.2.1
+psycopg2==2.8.2 --no-binary psycopg2
```

Comment:
It is possible that the `--no-binary` option is no longer necessary post 2.7.x.

See: http://initd.org/psycopg/docs/install.html

Reply:
Nice find. Good instinct to look a bit further too.